### PR TITLE
Remove Travis Build Badge

### PR DESCRIPTION
**Pull request by mikaelm12**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Add the VSTS badge
[Build Status](https://dnceng.visualstudio.com/public/_build/latest?definitionId=26)

**Comment by BrennanConroy** [comment]
Forgive me, but these would be nice too...

[npm version](https://badge.fury.io/js/%40aspnet%2Fsignalr)[NuGet version](https://badge.fury.io/nu/microsoft.aspnetcore.signalr)

File: README.md [hunk 1]
```diff
@@ -3,9 +3,11 @@ ASP.NET Core SignalR
 
 **IMPORTANT**: This repository hosts code and project management for ASP.NET **Core** SignalR, for use in ASP.NET Core applications using `Microsoft.AspNetCore.App`. If you are looking for information on ASP.NET SignalR (used in .NET Framework applications using System.Web and/or Katana), see the https://github.com/SignalR/SignalR repository.
 
-[![Join the chat at https://gitter.im/aspnet/SignalR](https://badges.gitter.im/aspnet/SignalR.svg)](https://gitter.im/aspnet/SignalR?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/aspnet/SignalR/SignalR-ci)](https://dnceng.visualstudio.com/public/_build/latest?definitionId=26)
```
[the remark this answers is not in the document]
Comment (BrennanConroy) [reply README.md]: Thats not the one I was concerned about :P

[the remark this answers is not in the document]
Comment (anurse) [reply README.md]: The `public` project is designed for public read-only access.

**Pull request merged**